### PR TITLE
U/peter matias/astrometry precision

### PR DESCRIPTION
**Pull request by yoachim**
Super.

### CODE REVIEW

**Comment by cwwalter** [comment]
You accidentally modified a some other notebooks when the kernel changed.

File: matching.ipynb [hunk 1]
```diff
@@ -376,9 +376,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "LSST_Stack (Python 3)",
+   "display_name": "LSST",
```
Comment (cwwalter) [inline matching.ipynb]: This is stupid.

(this is a test)

File: round_trip_astrometry.ipynb [hunk 2]
```diff
@@ -444,9 +444,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "LSST_Stack (Python 3)",
+   "display_name": "LSST",
```
Comment (cwwalter) [inline round_trip_astrometry.ipynb]: This is stupid too.

**Comment by drphilmarshall** [comment]
PR needs an informative title :-P

**Review by cwwalter — COMMENTED** [review]
We can't understand the matching code pasta that was copied from another notebook.  Either explaining or putting in more description that you don't understand it would be good so the reader is warned first.

**Comment by cwwalter** [comment]
A description of what Multimatch does and that it adds and "object" would be useful.

**Review by cwwalter — APPROVED** [review]
I approve!

**Pull request merged**